### DISCUSSION TITLE
supabase_admin role

### DISCUSSION THREAD

**Discussion by mohammedabboudi** [supabase/supabase#16376] [Questions]
Hello there, I need help with executing the query provided on github for the database So please if someone can provide assistance I would appreciate it, when I'm trying to execute the query I get this error "Failed to run sql query: must be member of role "supabase_admin""

**Comment by GaryAustin1** [comment]
Not sure if your case is covered in this: https://github.com/orgs/supabase/discussions/9314  (many different cases discussed)

_Supabase managed schemas are used to support platform features for all projects. Entities in these schemas are owned by supabase_admin role to prevent users from accidentally overriding them and breaking platform features. Unless explicitly granted, non-superuser roles cannot manage entities in Supabase managed schemas after the migration._
